source/common/http/conn_pool_base.cc

```diff
@@ -84,6 +84,8 @@ HttpConnPoolImplBase::newPendingStream(Envoy::ConnectionPool::AttachContext& con
 void HttpConnPoolImplBase::onPoolReady(Envoy::ConnectionPool::ActiveClient& client,
                                        Envoy::ConnectionPool::AttachContext& context) {
   ActiveClient* http_client = static_cast<ActiveClient*>(&client);
+  // Initialize upstream network read filters, if any
+  http_client->codec_client_->initializeReadFilters();
```

Comment:
should we error handle if any fail?

Reply:
`initializeReadFilters()` returns `false` only if there are no read filters. Not sure of the current state, but maybe it is usual to not have any upstream (read) filters?